VERSION

```diff
@@ -1 +1 @@
-0.8.0
+0.8.1
```

Comment:
@Tomcli -- even though I have to bump up the SDK version to `0.8.1` for PyPI, we do not need (may not want) to bump the `VERSION` of the entire project to `0.8.1` which would/should require a new Git tag and GitHub release, along with new manifests, etc.

Reply:
@Tomcli -- so this change I will undo and keep the overall project version at `0.8.0` (also no new git tag, GitHub release), agreed?

Reply:
Reverted the `VERSION` back to `0.8.0`